go-controller/pkg/ovn/namespace_test.go

```diff
@@ -67,8 +67,8 @@ func newNamespace(namespace string) *v1.Namespace {
 	}
 }
 
-func getNsAddrSetHashNames(ns string) (string, string) {
-	return addressset.GetHashNamesForAS(getNamespaceAddrSetDbIDs(ns, DefaultNetworkControllerName))
+func getNsAddrSetHashNames(ns, controllerName string) (string, string) {
```

Comment:
I would say let's leave this function as is, and make a new one like `getMultinetNsAddrSetHashNames(ns, controllerName string)`, and it will have only 1 usage from `getGressACLs`, and will reduce the number of required changes